gitlens/gitlens-settings.md

```diff
@@ -305,6 +305,10 @@ GitLens is highly customizable and provides many configuration settings to allow
 <td><code>gitlens.views.searchAndCompare.files.icon</code></td>
 <td>Specifies how the Search & Compare view will display file icons</td>
 </tr>
+<tr>
+<td><code>gitlens.views.openOnlyChangedFiles</code></td>
+<td>TODO: needs description</td>
```

Comment:
@d13 @axosoft-ramint do either of you have a good description for this setting? I was unable to locate more information than what is included in the release notes

Reply:
This isn't a setting - it's a command. The command shows up in the command palette as `GitLens: Open Changed & Close Unchanged Files`. It is a combination of the `Open Changed Files` and `Close Unchanged Files` commands.